app/src/NativeBase/Containers/MyExperienceContainer.tsx

```diff
@@ -0,0 +1,117 @@
+/**
+ * @file the MyExperience screen for the STA Volunteer App
+ */
+import {
+  Heading,
+  VStack,
+  HStack,
+  Checkbox,
+  ScrollView,
+  Flex,
+  Button,
+  Box,
+  View,
+} from 'native-base'
+import StaRibbon from '../Components/StaRibbon'
+import StaThemeLogo from '@/NativeBase/Assets/Images/Logos/sta-ribbon-logo.svg'
+import FreeSearchBar from '../Components/FreeSearchBar'
+import React from 'react'
+import { RoleGroup, roleGroups } from '@/Services/modules/projects/roleGroups'
+
+const MyExperienceContainer = () => {
+  const [skillsValue, setSkillsValue] = React.useState<string[]>([])
+  const [searchTxt, setSearchTxt] = React.useState<string>('') // search text
+
+  /**
+   * Filter the role groups based on the search text entered by the user
+   */
+  const filteredSkills = roleGroups.filter((roleGroup: RoleGroup) => {
+    return roleGroup.groupName.toLowerCase().includes(searchTxt.toLowerCase())
+  })
+
+  return (
+    <>
+      <HStack mt={-10} mb={12} alignItems={'flex-end'}>
+        <Heading
+          size="md"
+          letterSpacing={3}
+          fontWeight={400}
+          mb={-3}
+          fontFamily="BebasNeue-Regular"
+          adjustsFontSizeToFit
+        >
+          My Experience
+        </Heading>
+        <Flex flex={1} alignItems={'flex-end'} mt={-10}>
+          <StaThemeLogo width={'100%'} />
+        </Flex>
+      </HStack>
+      <StaRibbon />
+      <View margin={'4'}>
+        <FreeSearchBar
+          marginTop="2"
+          handleSubmit={() => {
+            null
+          }}
+          handleChangeText={(text: string) => {
+            setSearchTxt(text)
+          }}
+        />
+      </View>
+      <ScrollView>
+        <VStack mt={-3} margin="2">
+          <Checkbox.Group
+            ml="2"
+            onChange={(value: string[]) => {
```

Comment:
Similar point to above -- could simplify this into a one-liner without curly brackets around the function